internal/services/oracle/oracle_exascale_database_virtual_machine_cluster_resource_test.go

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package oracle_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-09-01/exadbvmclusters"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/oracle"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type ExascaleDatabaseVirtualMachineClusterResource struct{}
+
+func (a ExascaleDatabaseVirtualMachineClusterResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := exadbvmclusters.ParseExadbVMClusterID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := client.Oracle.OracleClient.ExadbVMClusters.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving %s: %+v", id, err)
+	}
+	return pointer.To(resp.Model != nil), nil
+}
+
+func TestExascaleDatabaseVirtualMachineClusterResource_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, oracle.ExascaleDatabaseVirtualMachineClusterResource{}.ResourceType(), "test")
+	r := ExascaleDatabaseVirtualMachineClusterResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
```

Comment:
The test needs to include coverage for computed fields.
```suggestion
				check.That(data.ResourceName).ExistsInAzure(r),
				check.That(data.ResourceName).Key("ocid").Exists(),
				check.That(data.ResourceName).Key("zone_ocid").Exists(),
```